src/woo_publications/metadata/api/views/category.py

```diff
@@ -0,0 +1,29 @@
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import viewsets
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin
+
+from woo_publications.metadata.models import InformationCategory
+
+from ..filterset.category import InformationCategoryFilterset
+from ..serializer.category import InformationCategorySerializer
+
+
+@extend_schema(tags=["informatie categorie"])
+@extend_schema_view(
+    list=extend_schema(
+        summary=_("Alle informatie categorieën opvragen."),
+        description=_("Alle informatie categorieën opvragen."),
+    ),
+    retrieve=extend_schema(
+        summary="Een specifiek informatie categorie opvragen.",
+        description="Een specifiek informatie categorie opvragen.",
+    ),
```

Comment:
Please do these in English, we will add translations later.

At any given time - marking a Dutch string as translatable is weird, the source language is English and we create a Dutch translation file to enter the Dutch strings. I don't want to introduce any inconsistency in those files.